src/TRestTrackLinearizationProcess.cxx

```diff
@@ -103,7 +103,7 @@ TRestEvent* TRestTrackLinearizationProcess::ProcessEvent(TRestEvent* evInput) {
         fOutTrackEvent->AddTrack(fTrackEvent->GetTrack(t));
 
     for (int t = 0; t < fTrackEvent->GetNumberOfTracks(); t++) {
-        if (!fTrackEvent->isTopLevel(t)) continue;
+        if (fTrackEvent->GetLevel(t) != 1) continue;
```

Comment:
Why `!= 1`? Should not be` > 0`? If a track is level 2 it will not continue.

Reply:
This is the way that I found to make it work, it seems that the lowest track level is 1 instead of 0. If you want I can change the statement to `if (fTrackEvent->GetLevel(t) > 1) continue;`

Reply:
Yes, I think it should be `>` in case in the future another process plugged just before increases the number of levels. This way you are sure the processing only affects to the lower track level.

Reply:
Ok, I see `!=` is also valid, but for some reason for me it is easier to understand because I have to check mentally the behaviour only at the right of 1.